DataFormats/Common/src/classes_def.xml

```diff
@@ -204,4 +204,8 @@
   <version ClassVersion="3" checksum="2237762164"/>
  </class>
  <class name="edm::Wrapper<std::map<edm::ParameterSetID,edm::ParameterSetBlob>>"/>
+
+ <class name="edm::StdArray<short, 4>" ClassVersion="10">
```

Comment:
For new classes the versioning should start from 3
```suggestion
 <class name="edm::StdArray<short, 4>" ClassVersion="3">
```

Reply:
I thought we were supposed to start from 10 ?
OK, I'll use 3.

Reply:
Class version 10 is for classes that have been persisted before without an explicit class version (sort of a guess that hopefully there have been less than ~8 different versions of it in the past).

Reply:
Ah, ok.